### PR TITLE
EDU-1094 Add initial volume to media plugins

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1094

**Please review [EDU-1091 ](https://github.com/educandu/educandu/pull/832) first.** This branch includes the other one, due to overlapping changes.

With this ticket I also:
- improved the look of the timeline (in `interactive-media` and `media-analysis`), when in the initial state:

Screen Shot 2023-02-03 at 17 00 55

- improved the `ear-training` plugin look:

BEFORE
Screen Shot 2023-02-03 at 17 05 50

AFTER
Screen Shot 2023-02-03 at 17 02 24

- improved the preview and track mixer look in `multitrack-media` and `media-analysis` to make it clearer to the user what is "settable" and what is just "playable":

BEFORE
Screen Shot 2023-02-03 at 17 06 16

AFTER
Screen Shot 2023-02-03 at 17 04 43

### CODE REVIEW

File: src/components/media-player/timeline.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+timeline:
+  markersBarPlaceholder:
+    en: Hover to see marker
+    de: Bitte hover to see marker
```
Comment (ahelmberger) [inline src/components/media-player/timeline.yml:4]: Bewege die Maus über den Streifen, um Markierungen zu setzen

File: src/components/media-player/track-mixer-editor.yml [hunk 2]
```diff
@@ -2,6 +2,9 @@ trackMixerEditor:
   noTrack:
     en: no track
     de: keine Spur
-  manageVolumePresets:
-    en: Manage volume presets
-    de: Lautstärke-Voreinstellungen bearbeiten
+  volumePresetInfo:
+    en: Sets of track volume combinations. Adjust the track volumes to set up the current preset.
+    de: Sets of track volume combinations, ja? :)
```
Comment (ahelmberger) [inline src/components/media-player/track-mixer-editor.yml:7]: Hier kannst du Sets von Lautstärke-Kombinationen definieren. Die einzelnen Spurlautstärken der momentan ausgewählten Kombination kannst im Spur-Mixer unten bearbeiten.

File: src/resources/common.yml [hunk 3]
```diff
@@ -489,6 +486,15 @@ common:
...
     en: Filter
     de: Filter
+  playerSettings:
+    en: Player settings
+    de: Player-Einstellungen
+  initialVolume:
+    en: Initial volume
+    de: Anfangslautstärke
+  multitrackInitialVolumeInfo:
+    en: This is the initial volume of the player, applied over all tracks. Available only when there is more than one track. [but better]
```
Comment (ahelmberger) [inline src/resources/common.yml:496]: This is the initial mix volume

Dies ist die initiale Mix-Lautstärke

**Review by ahelmberger — APPROVED** [review]

**Comment by ahelmberger** [comment]
> AFTER Screen Shot 2023-02-03 at 17 04 43

Love it!

File: src/components/media-player/timeline.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/media-player/timeline.yml]: Changed!

File: src/components/media-player/track-mixer-editor.yml (hunk 2 above)
Comment (adrianaluchian) [reply src/components/media-player/track-mixer-editor.yml]: Translated this in English too, thanks! 👍

File: src/resources/common.yml (hunk 3 above)
Comment (adrianaluchian) [reply src/resources/common.yml]: Changed!

**Comment by adrianaluchian** [comment]
> 
> Love it!

Great ❤️

**Pull request merged**